### ISSUE TITLE
bootstrap - easy pie chart conflict

### ISSUE THREAD

**Issue by developerchil** [developerchil/hubble-landing-page-v2#2]
Hello gays!
I try to embed easy pie chart.js in my bootstrap page, but something gets wrong always and I can not realize what it's and how to fix it.
Please, help
Here is my HTML code: <head>
    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" >
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" class="rel">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.0/jquery.min.js"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js"></script> 
    
    <link rel="icon" type="image/png" href="images/favicon.ico">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/assets/owl.carousel.min.css" class="rel">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/superslides/0.6.2/stylesheets/superslides.css" class="rel">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/fancyapps/fancybox@3.5.7/dist/jquery.fancybox.min.css" />
    <link rel="stylesheet" href="style.css" class="rel">    
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css">
    <link rel="stylesheet" href="styles.css" /> 
    
          
    <title>Document</title>
</head>
<body>

<nav class="navbar navbar-expand-lg navbar-dark bg-dark mt-2">
  <a class="navbar-brand" href="#">BootChilPage</a>
  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>

  <div class="collapse navbar-collapse" id="navbarSupportedContent">
    <ul class="navbar-nav mr-auto">
      <li class="nav-item active">
        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#about">About</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#skills">Skills</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#">Gallery</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#">Statistic</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#">Portfolio</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#contact">Contact</a>
      </li>      
    </ul>    
  </div>
</nav>

<div id="main-container" class="container mt-2">
  <div class="center">  
   <p class="sub typed"></p>
  </div>
</div> 

<div id="about" class="container-fluid">
  <div class="row">
    <div class="col-xs-12 col-sm-6 col-md-5 col-lg-6 col-xl-6">
      <h4 class="mt-5 text-center">I have known as <a id="gray" href="#">DevChil</a> ...</h4>
      <p class="no-bgd text-center mt-5 ml-5">No, I am not so anonymous...I just like a lot my nickname:)</p>
      <p class="no-bgd text-justify mt-5 ml-5">All the Lorem Ipsum generators on the Internet tend to repeat predefined chunks as necessary, making this the first true generator on the Internet.
      It uses a dictionary of over 200 Latin words. 
      <p class="no-bgd text-justify mt-5 ml-5 mb-5">The generated Lorem Ipsum is therefore always free from repetition, injected humour, or non-characteristic words etc.combined with a handful of model sentence structures, to generate Lorem Ipsum which looks reasonable.</p>
    </div>
    <div class="col-xs-12 col-sm-6 col-md-7 col-lg-6 col-xl-6">
      <img src="images/code.jpg" class="right-side-img col-lg-4 col-xl-6 col-md-6 col-sm-6 col-xs-12 float-right" alt="programming-learn"/>
    </div>
  </div>
</div>

<div id="skills" class="container-fluid">
  <div class="row">
    <div class="col-md-12 text-center">
      <h2 class="tech mt-4">Techical skills</h2>
      <p class="tech-2">Here is a presentation of my front-end skills used in this project</p>
      <p class="tech-2">Refresh the page to see the effect</p>
      <p class="white-line"></p>

      <div class="box">
        <div class="chart mt-4" data-percent="93">93%
          <h5>HTML5</h5>
        </div>
      </div>

      <div class="box">
        <div class="chart mt-4" data-percent="90">90%
          <h5>CSS3</h5>
        </div>
      </div>

      <div class="box">
        <div class="chart mt-4" data-percent="68">68%
          <h5>jQuery</h5>
        </div>
      </div>

      <div class="box">
        <div class="chart mt-4" data-percent="72">82%
          <h5>Bootstrap4</h5>
        </div>
     </div>
     
   </div>
 </div>
</div>

   
               

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/easy-pie-chart/2.1.6/jquery.easypiechart.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/owl.carousel.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/typed.js/2.0.10/typed.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/superslides/0.6.2/jquery.superslides.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
 
<script src="https://cdn.jsdelivr.net/npm/typed.js@2.0.9"></script> 
  
<script src="JS/boot.js"></script>                                     

                    
</body>

here is my CSS: @import url('https://fonts.googleapis.com/css?family=Montserrat|Rokkitt');
@import url('https://fonts.googleapis.com/css?family=Josefin+Slab:600');
@import url('https://fonts.googleapis.com/css?family=Abril+Fatface|Montserrat');
@import url('https://fonts.googleapis.com/css?family=Raleway');

html {
    scroll-behavior: smooth;
}

 body {
     background-color: #E1DCD9;
     font-family: 'Rokkitt', serif;
}

.navbar {
    background: maroon !important;
    width: 95%;
    margin: 0 auto;
}

.navbar-brand {
    color: white;
    letter-spacing: 4px;
    text-shadow: 2px 2px 5px black;
    font-weight: bold;
    font-size: 1.75rem; 
}

.nav-link {
    color: white;
    font-size: 1.30rem;
    font-family: 'Rokkitt', serif;
}

.navbar-brand {
   font-family: 'Josefin Slab', serif;
   font-weight: bold;
}

#main-container {
    background-image: url("images/working-desk.jpg");
    background-size: cover;
    background-repeat: no-repeat;
    //position: center center;
    min-height: 650px;
    max-width: 95%;
    border: 2px solid black;
}

#main-container img {
    max-width: 100%;
}

#main-container .sub {
    position: absolute;
    top:70%;
    left:50%;
    transform: translate(-50%, -50%);
    width: 95%;
    text-align: center;
    color:white;
    text-shadow: 2px 2px 2px black 1px 1px 1px maroon;
    margin: 0;
    padding: 0;
    font-size: 2.0rem;
    letter-spacing: 2px;
    font-weight: bold;
    text-transform: uppercase;
}

#main-container h2 span {
    color: maroon;
}

.container-fluid {
    background-color: #082d57e6;
    min-height: 400px;
    max-width: 95%;
    margin-top:60px;
}

.container-fluid h4, p {
    color:white;
    background-color: transparent;
    text-shadow: 2px 2px 1px maroon;
}

.container-fluid h4 {
    font-size: 28px;
}

.container-fluid p {
    font-size:18px;
    letter-spacing: 2px;
}

#gray {
    color: maroon;
    text-shadow: 1px 2px 4px white;
    font-weight: bold;
}
.container-fluid img {
    max-width: 90%;
    padding: 5px;
    border: 1px solid white;
    margin: 16px; 
}

#skills {
    background-color: maroon;
    min-height: 400px;
    max-width: 95%;
    margin-top:60px;
}

.white-line {
    height: 4px;
    color: white;
    border-bottom: 1px solid white;
}

#skills h2 {
    color: white;
    font-size:40px;
    text-shadow: 1px 2px 1px black;
    letter-spacing: 2px
}

.tech-2 {
    letter-spacing: 3px;
}

.box {
    width: 25%;
    float: left;
    color: white;
}

.chart {
    position: relative;
    width: 110px;
    height: 110px;
    margin: 0 auto;
    text-align: center;
    font-size: 28px;
}

.mySlides {display:none;}

.footer {
    width: 95%;
    margin: 0 auto;
    background: black;
    height: 120px;
}
.footer p {
    color: white;
    font-size: 22px;
}

.rights {    
    text-align: center;
    padding: 22px 0;
    font-weight: bold;
    letter-spacing: 2px;
}

.rights a {
    color: maroon;
    text-shadow: 1px 1px 1px white;
    font-size: 1.50rem;
    letter-spacing: 3px;
}
.fab {
    margin-right: 16px;
    color: maroon;
    text-decoration: none;
    text-shadow: none;
    font-size: 22px;
    background: #E1DCD9;
    opacity: 0.5;
    padding: 6px;
    border-radius: 4px;
    transition: 0.3s ease;
    width: 55px;
}

.fab:hover {
    opacity: 1;
    
}

.footer span {
    line-height: 60px;
}

here is my JS:   var typed = new Typed(".typed", {
        strings: [
            "I am DevChil",
            "I am  a Front End Developer",
            "Welcome to My Bootstrap page challenge "
        ],
        typeSpeed: 50,
        loop: true,
        startDelay: 1000,
        showCursor: false

    });

  var element = document.querySelector('.chart');
    new EasyPieChart(element, {
        // your options goes here
    });
